src/index.tsx

```diff
@@ -79,7 +79,7 @@ class ReactInlineSVG extends React.PureComponent<Props, State> {
       }
     }
 
-    if (previousProps.src !== src) {
+    if (previousProps !== this.props) {
```

Comment:
This will run every time since `props` is an object and can't be compared with strict equality.